README.md

```diff
@@ -2,92 +2,707 @@
+
+For more information about Pelican, see our [main website](https://pelicanplatform.org) or [GitHub page](https://github.com/PelicanPlatform/pelican). For more information about fsspec, visit the [filesystem-spec](https://filesystem-spec.readthedocs.io/en/latest/index.html) page.
+
+For comprehensive tutorials and real-world examples using PelicanFS with geoscience datasets, see the [Project Pythia OSDF Cookbook](https://projectpythia.org/osdf-cookbook/).
+
+**Note on Terminology:**
+- In URL terminology, `pelican://` and `osdf://` are properly called **schemes**. While fsspec refers to these as "protocols," we use the term "scheme" throughout this documentation for technical accuracy.
+- Pelican is an **object storage** system. Remote items are called **objects** (analogous to files) and **collections** (analogous to directories), not files and directories.
 
+## Features
+
+- **Read Operations**: List, read, and search for objects across Pelican namespaces
+- **Write Operations**: Upload objects to Pelican origins with proper authorization
+- **Smart Caching**: Automatic cache selection and fallback for optimal performance
+- **Token Management**: Automatic token discovery and validation for authorized operations
+- **Scheme Support**: Works with both `pelican://` and `osdf://` URL schemes
+- **Integration**: Seamless integration with popular data science libraries (xarray, zarr, PyTorch, etc.)
+- **Async Support**: Built on async foundations for efficient I/O operations
 
 ## Limitations
 
-PelicanFS is built on top of the http fsspec implementation. As such, any functionality that isn’t available in the http implementation is also *not* available in PelicanFS.
+PelicanFS is built on top of the HTTP fsspec implementation. As such, any functionality that isn't available in the HTTP implementation is also *not* available in PelicanFS. Specifically:
+- `rm` (remove files)
+- `cp` (copy files)
+- `mkdir` (create directories)
+- `makedirs` (create directory trees)
+
+These operations will raise `NotImplementedError` if called.
 
 ## Installation
 
-To install pelican, run:
+To install PelicanFS from PyPI:
 
-```
+```bash
 pip install pelicanfs
 ```
 
-To install from source, run:
+To install from source:
 
-```
+```bash
 git clone https://github.com/PelicanPlatform/pelicanfs.git
 cd pelicanfs
 pip install -e .
 ```
 
+## Quick Start
 
-## Using PelicanFS
+### Basic Usage
 
-To use pelicanfs, first create a `PelicanFileSystem` and provide it with the pelican federation url. As an example using the OSDF federation
+Create a `PelicanFileSystem` instance and provide it with your federation's discovery URL:
 
 ```python
 from pelicanfs.core import PelicanFileSystem
 
+# Connect to the OSDF federation
 pelfs = PelicanFileSystem("pelican://osg-htc.org")
+
+# List objects in a namespace
+objects = pelfs.ls('/ospool/uc-shared/public/OSG-Staff/')
+
+# Read an object
+content = pelfs.cat('/ospool/uc-shared/public/OSG-Staff/validation/test.txt')
```

Comment:
This example should point at our canonical test object -- `/pelicanplatform/test/hello-world.txt`